### PR TITLE
[wip] cleanup local dev ux

### PR DESCRIPTION
**Pull request by detiber**
WIP refactor of some of the local dev experience

Example run from my fork (fails on cosign): https://github.com/detiber/chonky-main/actions/runs/12405816581/job/34633290193

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -36,68 +39,60 @@ jobs:
...
-          echo "alias_tags=${alias_tags}" >> $GITHUB_OUTPUT
-
-      # Tag Images
-      - name: Tag Images
-        shell: bash
-        run: |
-          set -eoux pipefail
-          just tag-images "${{ env.IMAGE_NAME }}" \
-                          "${{ env.DEFAULT_TAG }}" \
-                          "${{ steps.generate-tags.outputs.alias_tags }}"
```
Comment (detiber) [inline .github/workflows/build.yml:100]: Pulled this back into the github action workflow, since it's not needed for local dev.

File: Justfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
-export repo_organization := "centos-workstation"
-export image_name := "main"
-export centos_version := "stream10"
+export repo_organization := env("GITHUB_REPOSITORY_OWNER", "centos-workstation")
+export image_name := env("IMAGE_NAME", "main")
+export centos_version := env("CENTOS_VERSION", "stream10")
+export default_tag := env("DEFAULT_TAG", "latest")
```
Comment (detiber) [inline Justfile:4]: streamline variable config between what's being set in github actions and here.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -107,6 +102,13 @@ jobs:
         with:
           string: ${{ env.IMAGE_REGISTRY }}
 
+      - name: Build Image
+        id: build-image
+        shell: bash
+        run: |
+          IMAGE_FULL="${{ steps.registry_case.outputs.lowercase }}/${IMAGE_NAME}"
+          just build "${IMAGE_FULL}" "${DEFAULT_TAG}"
```
Comment (tulilirockz) [inline .github/workflows/build.yml:110]: We need to refactor the build-vm and run-vm ones after this gets merged. This will kind of break it I think..?

Comment (detiber) [reply .github/workflows/build.yml:110]: That's the main reason this is still wip, but wanted to get this pushed before I called it a night

Comment (tulilirockz) [reply .github/workflows/build.yml:110]: Worst case dont worry we can work on it outside of this PR. I want to keep the achillobator `just run-vm 1 raw` workflow, it is super cleeeean!

Comment (detiber) [reply .github/workflows/build.yml:110]: build-vm and run-vm recipes are working now. I did refactor run-vm to run in a container (blatantly stolen from the bluefin run-iso job)

**Review by castrojo — APPROVED** [review]

**Pull request merged**